Capstone/API/APIManager.m

```diff
@@ -37,8 +37,10 @@ + (instancetype)shared {
     return sharedManager;
 }
 
-- (void)fetchCollegeNews:(void(^)(NSArray *collegeNews, NSError *error))completion {
-    NSURL *url = [NSURL URLWithString:@"https://newsapi.org/v2/everything?q=University&apiKey=7f0d9c3ee4ac401e8d6a714629947c61"];
+- (void)fetchCollegeNews:(NSString *)collegeWebsite getArrayOfColelgeNews:(void(^)(NSArray *collegeNews, NSError *error))completion {
```

Comment:
```suggestion
getArrayOfCollegeNews

Reply:
Since we're inputting domains of queried colleges using "(NSString *)collegeWebsite" isn't better to continue working with the previous code?